node/pkg/driver/node_test.go

```diff
@@ -1138,74 +1139,105 @@ func TestNodeGetInfo(t *testing.T) {
 	topologySegments := map[string]string{"topology.block.csi.ibm.com/zone": "testZone"}
 
 	testCases := []struct {
-		name              string
-		return_iqn        string
-		return_iqn_err    error
-		return_fcs        []string
-		return_fc_err     error
-		return_nodeId_err error
-		expErr            error
-		expNodeId         string
-		iscsiExists       bool
-		fcExists          bool
+		name            string
+		returnNqn       string
+		returnNqnErr    error
+		returnFcs       []string
+		returnFcErr     error
+		returnIqn       string
+		returnIqnErr    error
+		returnNodeidErr error
+		expErr          error
+		expNodeId       string
+		nvmeExists      bool
+		fcExists        bool
+		iscsiExists     bool
 	}{
 		{
-			name:          "good iqn, empty fc with error from node_utils",
-			return_fc_err: fmt.Errorf("some error"),
-			expErr:        status.Error(codes.Internal, fmt.Errorf("some error").Error()),
-			iscsiExists:   true,
-			fcExists:      true,
+			name:        "good iqn, empty fc with error from node_utils",
+			returnFcErr: fmt.Errorf("some error"),
+			expErr:      status.Error(codes.Internal, fmt.Errorf("some error").Error()),
+			nvmeExists:  false,
+			fcExists:    true,
+			iscsiExists: true,
+		},
+		{
+			name:         "empty nqn with error from node_utils, one more fc ports",
```

Comment:
> one more fc ports

I know this exists in existing code, but since you add it as well - what exactly does it mean (even when assuming "one" can describe multiple ports)?

Reply:
good point, done